ui/analyse/src/ctrl.ts

```diff
@@ -442,6 +442,12 @@ export default class AnalyseCtrl {
   userMove = (orig: Key, dest: Key, capture?: JustCaptured): void => {
     this.justPlayed = orig;
     this.justDropped = undefined;
+    if (!capture) {
+        var piece = this.chessground.state.pieces[dest];
+        if (piece && piece.role === 'pawn' && dest[0] != orig[0]) {
+            capture = piece;
+        }
+    }
     this.sound[capture ? 'capture' : 'move']();
     if (!promotion.start(this, orig, dest, capture, this.sendMove)) this.sendMove(orig, dest, capture);
```

Comment:
Thanks. Committed a similar fix (b7f1fe64b904dd22d8071f00aaef73fddda69396) that doesn't modify `capture` (or certainly doesn't set it to `piece`) because it is passed down to sendMove where it could interfere with Crazyhouse.